### PR TITLE
fix: overwrites contributions if user exists

### LINKED ISSUE #10: Bot currently overwrites contributions if user exists (does not add to them)
**Is your feature request related to a problem? Please describe.**
The bot's PR currently overwrites previous contributions (does not add to them)

**Describe the solution you'd like**
When a user already exists merge the contributions, instead of replacing them.

**Describe alternatives you've considered**

**Additional context**
#9

### PR DESCRIPTION
**Pull request by sinchang**
fixes #10

### CODE REVIEW

**Comment by sinchang** [comment]
> Looks great thanks for this! Any chance we could get a specific test in parse-comment too?

tests added. not sure if it is correct, welcome comments.

File: src/tasks/processIssueComment/probot-processIssueComment.js [hunk 1]
```diff
@@ -141,14 +152,18 @@ async function probotProcessIssueComment({ context, commentReply, analytics }) {
             branchName,
         })
         const optionsConfig = await setupOptionsConfig({ repository })
+        const oldContributions = findOldContributions(optionsConfig, safeWho)
```
Comment (jakebolam) [inline src/tasks/processIssueComment/probot-processIssueComment.js]: How do you feel about moving all this logic, into 
https://github.com/all-contributors/all-contributors-bot/blob/master/src/tasks/processIssueComment/OptionsConfig/index.js#L74

Would make it easier to unit test too

File: test/tasks/processIssueComment/probot-app-e2e.test.js [hunk 2]
```diff
@@ -22,6 +22,19 @@ describe('All Contributors app - End to end', () => {
         return true
     }
 
+    const verifyBodyForPull = body => {
```
Comment (jakebolam) [inline test/tasks/processIssueComment/probot-app-e2e.test.js]: See comment about, if we're able to move the logic into the OptionsConfig class we can just have an test on that class (no need for more e2e ones) thoughts?

**Comment by sinchang** [comment]
@jakebolam updated. thanks your comment.

**Pull request merged**